src/ui/public/timepicker/timepicker.less

```diff
@@ -0,0 +1,19 @@
+.kbn-timepicker {
+  [kbn-time-input] {
+    text-align: center;
+  }
+  .kbn-timepicker-modes {
+    text-transform: capitalize;
+  }
+  .kbn-timepicker-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-refresh-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-timepicket-alert {
+    width: 100px;
+  }
+}
```

Comment:
Could you unnest these selectors please? The nesting adds additional specificity, but this level of specificity isn't needed.

```less
.kbn-timepicker [kbn-time-input] {
  text-align: center;
}
.kbn-timepicker-modes {
  text-transform: capitalize;
}
.kbn-timepicker-section {
  float: left;
  padding: 0px 15px;
}
.kbn-refresh-section {
  float: left;
  padding: 0px 15px;
}
.kbn-timepicket-alert {
  width: 100px;
}
```